### PR TITLE
A bunch of coverity fixes

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: src/libsystemd-network/icmp6-util.c [hunk 1]
```diff
@@ -162,7 +162,7 @@ int icmp6_receive(int fd, void *buffer, size_t size, struct in6_addr *ret_dst,
         };
         struct cmsghdr *cmsg;
         struct in6_addr addr = {};
-        triple_timestamp t;
+        triple_timestamp t = {};
```
Comment (yuwata) [inline src/libsystemd-network/icmp6-util.c:165]: Oh, thank you.

**Review by yuwata — APPROVED** [review]
LGTM.

File: src/core/execute.c [hunk 2]
```diff
@@ -3202,11 +3202,16 @@ static int apply_mount_namespace(
 
         if (MANAGER_IS_SYSTEM(u->manager)) {
                 propagate_dir = path_join("/run/systemd/propagate/", u->id);
-                if (!propagate_dir)
-                        return -ENOMEM;
+                if (!propagate_dir) {
```
Comment (bluca) [inline src/core/execute.c:3205]: whoops, missed that we had a cleanup to do

**Review by bluca — APPROVED** [review]
lgtm

**Comment by keszybz** [comment]
bionic-i386 and bionic-s390x passed, let's merge.

**Pull request merged**